stablehlo/dialect/TypeInference.cpp

```diff
@@ -1783,6 +1783,105 @@ LogicalResult inferDynamicUpdateSliceOp(
   return success();
 }
 
+// We intend to verify the following properties
+// P1. 1 <= rank <= 3
+// P2. Element types agree with fft_type
+// P3. Operand shape dimensions agree with fft_length for the given fft_type
+LogicalResult inferFftOp(
+    Optional<Location> location, Value operand, bool isRfft, bool isIrfft,
+    DenseIntElementsAttr fftLength,
+    SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes) {
+  auto fftLengthes = fftLength.getValues<int64_t>();
```

Comment:
How about `fftLengthValues`? This is more verbose but avoids confusion between two very similarly named variables.

Reply:
Agreed.